### ISSUE TITLE
Can't resolve 'mobx' in '.../node_modules/mobx-react' after install in a fresh react app

### ISSUE THREAD

**Issue by jcollum-nike** [mobxjs/mobx-react#506]
`Module not found: Can't resolve 'mobx' in '/Users/jco/my-app/node_modules/mobx-react'`

Steps: 
1. follow guide here: 
https://github.com/facebook/create-react-app/blob/master/README.md#getting-started

1. yarn start

1. confirm app works (it does)

1. yarn add mobx-react

1. add `import { observer } from 'mobx-react';` to `src/App.js`

Result: 

```
Failed to compile
./node_modules/mobx-react/index.module.js
Module not found: Can't resolve 'mobx' in '/Users/jcol53/learning/my-app/node_modules/mobx-react'
This error occurred during the build time and cannot be dismissed.
```

This could be a react issue but the [relevant issue is locked](https://github.com/facebook/create-react-app/issues/2534). 

Should I post this in the React repo instead?

**Comment by kikoseijo** [comment]
Add it.

yarn add mobx-react mobx-persist mobx mobx-react-form validatorjs valid-url

**Comment by jcollum-nike** [comment]
That yarn add seems to have fixed it. Is this a bug?

**Comment by mweststrate** [comment]
No, mobx is a peer dependency. It is like using react-dom without react

Op vr 15 jun. 2018 23:58 schreef jcollum-nike <notifications@github.com>:
